src/main.rs

```diff
@@ -187,15 +208,14 @@ fn main() {
 }
 
 #[cfg(feature = "debug_ctap")]
-fn print_packet_notice(notice_text: &str, timer: &Timer) {
-    let now = timer.get_current_clock().flex_unwrap();
-    let now_us = (Timestamp::<f64>::from_clock_value(now).ms() * 1000.0) as u64;
+fn print_packet_notice(notice_text: &str, clock: &LibtockClock) {
+    let now = clock.try_now().unwrap();
     writeln!(
         Console::new(),
         "{} at {}.{:06} s",
         notice_text,
-        now_us / 1_000_000,
-        now_us % 1_000_000
+        now / 1_000_000,
+        now % 1_000_000
```

Comment:
Did you try to deploy on the dev-kit with `debug_ctap`?

```
./deploy.py --board=nrf52840dk_opensk --opensk --panic-console --debug
```

And check the output in RTT when register+login in webauthn.io?

Reply:
I didn't find an easy way to print out the Instant. Therefore in the second commit I just remove the time printing. I'll try to find a way to print the Instant and push another commit.

Reply:
I updated the PR. PTAL.